notes/routes.py

```diff
@@ -0,0 +1,23 @@
+from fastapi import APIRouter, Depends
+from sqlalchemy.orm import Session
+from notes.models import Notes
+from notes.schemas import CreateNotes, NoteResponse
+from database.database import get_db # type: ignore
+from auth.routes import get_user
+from auth.models import Users
+from typing import List
+
+router = APIRouter()
+
+@router.get("/read-notes", response_model=List[NoteResponse])
```

Comment:
In Restful API design we don't use verbs in endpoint names as we have HTTP method to describe the action. So this endpoint should be notes. And read is implied from the GET http method.